### PR TITLE
[Applab Datasets] Add last updated time to library pane

### PR DESCRIPTION
**Pull request by ajpal**
1. Add the timestamp to the manifest when we run the live data cron jobs.
2. Show the last updated time using `moment.fromNow()`, which is translatable.
image

## Links

- [jira](https://codedotorg.atlassian.net/browse/STAR-613)

## Testing story

# Reviewer Checklist:

- [ ] Tests provide adequate coverage
- [ ] Code is well-commented
- [ ] New features are translatable or updates will not break translations
- [ ] Relevant documentation has been added or updated
- [ ] User impact is well-understood and desirable
- [ ] Pull Request is labeled appropriately
- [ ] Follow-up work items (including potential tech debt) are tracked and linked

### CODE REVIEW

File: apps/src/storage/dataBrowser/LibraryTable.jsx [hunk 1]
```diff
@@ -135,7 +143,8 @@ class LibraryTable extends React.Component {
 
 export default connect(
   state => ({
-    libraryManifest: state.data.libraryManifest || {}
+    libraryManifest: state.data.libraryManifest || {},
+    locale: state.pageConstants && state.pageConstants.locale
```
Comment (epeach) [inline apps/src/storage/dataBrowser/LibraryTable.jsx:147]: Is this always set? Is there a fallback in case it isn't?

**Review by epeach — COMMENTED** [review]
Is there anything that prevents a user from uploading a HUGE dataset?

Edit - Not sure what I was talking about here, ignore please!

File: apps/src/storage/dataBrowser/LibraryTable.jsx (hunk 1 above)
Comment (ajpal) [reply apps/src/storage/dataBrowser/LibraryTable.jsx:147]: I believe it's always set, but even if it's not, moment will just default to english if no locale is set, so we should be fine either way.

**Review by epeach — APPROVED** [review]
:tada:

**Pull request merged**